runtime/lua/vim/treesitter/languagetree.lua

```diff
@@ -551,7 +551,7 @@ local function add_injection(t, tree_index, pattern, lang, combined, ranges)
     t[tree_index][lang][pattern] = { combined = combined, regions = {} }
   end
 
-  table.insert(t[tree_index][lang][pattern].regions, ranges)
+  t[tree_index][lang][pattern].regions[#t[tree_index][lang][pattern].regions + 1] = ranges
```

Comment:
👎

Reply:
This could work if the performance benefit is worth it (IDK that it is).
```suggestion
  local regions = t[tree_index][lang][pattern].regions
  regions[#regions + 1] = ranges
```

Reply:
A single call to table.insert is not worth the change. So I reverted. The time of creation of the temporary variable is equivalent to the time of the call. So there's no point here